frontend/packages/console-app/src/components/modals/clone/clone-pvc-modal.tsx

```diff
@@ -139,9 +142,11 @@ const ClonePVCModal = withHandlePromise((props: ClonePVCModalProps) => {
             />
           </FormGroup>
           <AccessModeSelector
+            label={t('console-app~Access mode')}
```

Comment:
Do we ever use anything other than "Access mode" for the label? If not, do we need a `label` prop?